src/styles/app.css

```diff
@@ -0,0 +1,19 @@
+.app {
+  display: flex;
+  flex-direction: column;
+  background-image: url("../../public/nasa-photo-earth.jpeg");
+  background-size: cover;
+  background-position: center center;
+}
+
+.nasa-logo {
+  margin: auto;
+  height: 300px;
+  width: 500px;
+}
+
+@media screen and (max-width: 767px) {
+  .app {
+    flex-direction: column;
+  }
+}
```

Comment:
Youre already using flex-direction column as standard, so this rule isnt changing anything! Should remove this